docs/download/77.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: default
+title: ICU 77
+nav_order: 899
+has_children: false
+parent: Downloading ICU
+---
+
+<!--
+© 2025 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU 77
+
+ICU is the [premier library for software internationalization](https://icu.unicode.org/#h.i33fakvpjb7o),
+used by a [wide array of companies and organizations](https://icu.unicode.org/#h.f9qwubthqabj).
+
+## Not released yet!
+
+**ICU 77 is still under development.
+Once we have a release candidate, use it for testing, but do not use it in production!**
+
+## Release Overview
+
+* TODO -- below is old
+
+ICU 77 updates to
+[CLDR 47](https://cldr.unicode.org/downloads/cldr-47)
+(TODO: blog)
+locale data with new locales, and various additions and corrections.
+
+TODO: Highlights
+
+The Java technology preview implementation of the CLDR MessageFormat 2.0 specification has been updated to match the
+[CLDR 46.1 spec](https://www.unicode.org/reports/tr35/tr35-74/tr35-messageFormat.html).
+
+The C++ technology preview implementation of MessageFormat 2.0 is not yet quite up to date with CLDR 46.1.
+
+For more details, including migration issues, see below.
+
+Please use the [icu-support mailing list](https://icu.unicode.org/contacts) and/or [find/submit error reports](https://icu.unicode.org/bugs).
+
+### Version Number
+
+The initial release has library version number 77.1.
+
+* Release date: 2025-03-TODO
+  * [List of tickets fixed in ICU 77](https://unicode-org.atlassian.net/issues/?jql=project%20%3D%20ICU%20AND%20status%20%3D%20Done%20AND%20resolution%20in%20%28Fixed%2C%20%22Fixed%20by%20Other%20Ticket%22%29%20AND%20fixVersion%20%3D%2077.1%20ORDER%20BY%20component%20ASC%2C%20created%20DESC)
+
+If there are maintenance releases, they will be 77.2, 77.3, etc. (During ICU 77 development, the library version number was 77.0.x.)
+
+Note: There may be additional commits on the [maint/maint-77](https://github.com/unicode-org/icu/tree/maint/maint-77) branch that are not included in the prepackaged download files.
+
+## Common Changes
+
+* [CLDR 47](https://cldr.unicode.org/downloads/cldr-47)
+  (TODO: blog):
+  * No major data collection for existing locales; focus on bug fixes and structural improvements
+  * New regional variants: English in several European countries, and Cantonese in Macau (`yue_Hant_MO`)
+  * Improved RBNF (number spellout) and transliteration data
+  * TODO: review:
+  * Note: ICU 76 does not yet support `portion-per-1e9` (aka per-billion). (See [ICU-22781](https://unicode-org.atlassian.net/browse/ICU-22781))
+* Subtle segmentation changes to make ICU fully conform to Unicode 16
+  * Word break: Root tailoring of colon reverted, Swedish & Finnish tailorings removed ([ICU-22941](https://unicode-org.atlassian.net/browse/ICU-22941))
+    * These tailorings were introduced in ICU 72, but feedback has been negative
```

Comment:
Not sure what this is supposed to be telling me.

Reply:
I am trying to say that this has been a relatively recent thing which we are now reverting

Reply:
I guess it's the "fully conform to Unicode 16" thing that's throwing me.  We diverged from Unicode 16 and now we're going back into sync with it?  Or something else?

Reply:
The root rules want to match the Unicode segmentation specs.
Sometimes CLDR & ICU experiment with changes before we upstream them back into Unicode proper.
This is a change that hadn't even been discussed with the UTC, and feedback has been negative, and the UTC disagreed with upstreaming this change, so it was decided to revert it in CLDR & ICU.
We should have done this as part of the Unicode 16 update, but missed it.
See the 2024-03-11 design-wg meeting minutes.

(In that meeting, we also agreed to introduce segmentation profiles, but we haven't designed those yet.)